uaclient/api/u/pro/attach/auto/full_auto_attach/v1.py

```diff
@@ -0,0 +1,120 @@
+from typing import List, Optional
+
+from uaclient import actions, cli, contract, entitlements, messages
```

Comment:
I'd like to suggest that none of our API functionality imports from `uaclient.cli`. In the future, we want to do the opposite: CLI will leverage the API functions. So I'd suggest not importing this, and moving the imported function out of the CLI module - maybe to Entitlements?

Reply:
Nice suggestion. On it.